app/Models/Color.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * Class Color.
+ *
+ * @property int|null $r
+ * @property int|null $g
+ * @property int|null $b
+ */
+class Color extends Model
```

Comment:
The following properties are missing:

    @property int    $id
    @property bool   $is_main
    @property Photo  photo